packages/babel-cli/src/babel-external-helpers.js

```diff
@@ -24,4 +24,4 @@ commander.option(
 commander.usage("[options]");
 commander.parse(process.argv);
 
-console.log(buildExternalHelpers(commander.whitelist, commander.outputType));
+console.error(buildExternalHelpers(commander.whitelist, commander.outputType));
```

Comment:
This one shouldn't be `.error` :)

Reply:
Is this one actually emitting code? (I think part of the confusion in the babel codebase is the usage of `console` for emitting the compilation output. Would be clearer if there were a dedicated stream for the actual JS output instead of hijacking `console`)

Reply:
The `babel-external-helpers` command is specifically building some code and then dumping it to the console for piping, so this change would make the command non-functional.

> Would be clearer if there were a dedicated stream for the actual JS output

I don't think it makes much difference in this case.

Reply:
> I don't think it makes much difference in this case

Really? It would have directly avoided this issue here, where the console.log was erroneously switched to console.error :D (And eliminated the confusion that I personally had about confirming whether this line emitted code or something else.)

Reply:
I shall make `printOutput`, `printInfo` and `printDebug` functions?

Reply:
yeah lgtm